docs/book.toml

```diff
@@ -1,14 +1,12 @@
 [book]
-title = "The Miden protocol"
-authors = ["Miden team"]
-description = "Description and core structures for the Miden Rollup protocol"
+authors      = ["Miden team"]
```

Comment:
Nit: I think in our `Cargo.toml`s we seem to use `authors = ["Miden contributors"]`.